build/scripts/DownloadVCLibsDesktop.ps1

```diff
@@ -0,0 +1,27 @@
+[CmdLetBinding()]
+Param(
+    [string]$Platform = "x64",
+    [string]$OutputDirectory = "D:\WindowsAppSDK\build\scripts"
+)
+
+$ProgressPreference = "SilentlyContinue"
+$ErrorActionPreference = "Stop"
```

Comment:
Very nice, but you should also specify

```
Set-StrictMode -Version 3.0
```

I added that to DevCheck.ps1 recently and it spotted a problem, and stopped me from creating a couple new ones. Plus it's good practice so there's that too :P

This comment applies throughout